pygeo/mphys/mphys_dvgeo.py

```diff
@@ -136,21 +136,54 @@ def nom_add_point_dict(self, point_dict):
             self.nom_addPointSet(v, k)
 
     def nom_addGlobalDV(self, dvName, value, func, childIdx=None, isComposite=False):
+        """Add a global design variable to the DVGeo object. This is a wrapper for the DVGeo.addGlobalDV method.
+
+        Parameters
+        ----------
+        dvName : str
+            A unique name to be given to this design variable group
+
+        value : float, or iterable list of floats
+            The starting value(s) for the design variable. This
+            parameter may be a single variable or a numpy array
+            (or list) if the function requires more than one
+            variable. The number of variables is determined by the
+            rank (and if rank ==1, the length) of this parameter.
+
+        func : python function
+            The python function handle that will be used to apply the
+            design variable
+
+        childIdx : int, optional
+            The zero-based index of the child FFD, if this DV is for a child FFD.
+            The index is defined by the order in which you add the child FFD to the parent.
+            For example, the first child FFD has an index of 0, the second an index of 1, and so on.
+
+        isComposite : bool, optional
+            Whether this DV is to be included in the composite DVs, by default False
+
+        Raises
+        ------
+        RuntimeError
+            Raised if the underlying DVGeo object is not an FFD
+        """
```

Comment:
Do we want to start putting docstrings in the MPhys wrapper? We'd have to maintain the common attributes in two places then and that could get messy.

Reply:
At the very very least there should be a docstring that explains the MPhys-specific arguments to the function, for the remainder of the arguments it might be fine to point to the docstring for the original DVGeo function.